shared/cfg/guest-os/Linux/JeOS/22.x86_64.cfg

```diff
@@ -0,0 +1,25 @@
+- 22.x86_64:
```

Comment:
Oh we had `JeOS.20` I just never used it... Still I think we don't have to release each version.